projects/plugins/jetpack/modules/contact-form/admin.php

```diff
@@ -1398,6 +1406,48 @@ public function get_gdrive_export_section() {
 		</div>
 		<?php
 	}
+
+	/**
+	 * Ajax handler. Returns a payload with connection status and html to replace
+	 * the Connect button with the Export button. Export button is copy from $button_html seen on get_gdrive_export_section
+	 */
+	public function test_gdrive_connection() {
+		$post_data = wp_unslash( $_POST );
+		$user_id   = (int) get_current_user_id();
+
+		if (
+			! $user_id ||
+			empty( sanitize_text_field( $post_data[ $this->export_nonce_field_gdrive ] ) ) ||
+			! wp_verify_nonce( sanitize_text_field( $post_data[ $this->export_nonce_field_gdrive ] ), 'feedback_export' )
+		) {
+			wp_send_json_error(
+				__( 'You aren’t authorized to do that.', 'jetpack' ),
+				403
+			);
+
+			return;
+		}
+
+		require_once JETPACK__PLUGIN_DIR . '_inc/lib/class-jetpack-google-drive-helper.php';
```

Comment:
Would it be worth to safeguard this by adding a check if the class exists here? I can't see how an error could happen right now, but maybe it's better to be safe than sorry?

```suggestion
		if ( ! class_exists( 'Jetpack_Google_Drive_Helper' ) ) {
			require_once JETPACK__PLUGIN_DIR . '_inc/lib/class-jetpack-google-drive-helper.php';
		}
```

Reply:
Never hurts to be extra careful, thanks!